src/d3a/models/area/redis_dispatcher/market_event_dispatcher.py

```diff
@@ -21,6 +22,9 @@ def __init__(self, area, root_dispatcher, redis):
             MarketEvent.OFFER.value: Event(),
             MarketEvent.OFFER_DELETED.value: Event(),
             MarketEvent.OFFER_CHANGED.value: Event(),
+            MarketEvent.BID_TRADED.value: Event(),
```

Comment:
Shouldn't there be a MarketEvent for posting bid like `MarketEvent.BID` ?

Reply:
There is no BID event in the MarketEvent enum. There is no need for this event in the current implementation.